src/core/Elsa.Abstractions/Services/Models/LoadWorkflowSettingsContext.cs

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
I see already @sfmskywalker say: "Do we need this commented code?" :relaxed:

Reply:
I deleted that class. Well spotted.